modules/ROOT/pages/scheduler-concept.adoc

```diff
@@ -2,72 +2,213 @@
+* In a Mule runtime engine cluster or multi-worker CloudHub deployment, the Scheduler executes only on the primary node (that is, only in one Mule instance).
 
-== Example Projects
+== Architecture
 
-There are several example projects in Anypoint Exchange that you can open in Anypoint Studio to learn more about how to use the Scheduler component:
+Like other Mule message source components, a Scheduler is the first component in a Mule flow.
 
-* _Data Sync Using Watermarking and Batch_
-* _Import Contacts into Microsoft Dynamics CRM_
-* _Import Leads into Salesforce_
-* _Importing a CSV File into mongoDB_
-* _Importing an Email Attachment Using the IMAP Connector_
-* _Salesforce to MySQL Database Using Batch Processing_
-* _Sending a CSV File Through Email Using SMTP_
+image::scheduler-component-ex.png["Scheduler component in Mule flow"]
+
+The XML for the Scheduler consists of the following elements:
 
-To download and open an example project while you are in Anypoint Studio, click the Exchange icon in the upper-left corner. Then, in the window that opens, log into Anypoint Exchange and search on the name of the project.
+* `<scheduler />`: Top-level element of the Scheduler.
+* `<scheduling-strategy />`: Second-level element that accepts one of the following children:
+** `<fixed-frequency />`: For common scheduling frequencies, such as every second.
+** `<cron />`: For <<cron_expressions, cron expressions>> and time zone settings.
 
-== Fixed Frequency
+The Scheduler sets a scheduling strategy at a fixed frequency or an interval based on a cron expression:
 
-You can use the following parameters to set a Scheduler to trigger a flow at a regular intervals.
+* The fixed frequency setting in the following Scheduler executes every fifteen seconds:
++
+[source, xml]
+----
+<flow name="ex-fixed-frequency-flow" >
+  <scheduler doc:name="Scheduler">
+    <scheduling-strategy>
+      <fixed-frequency frequency="15" timeUnit="SECONDS" />
+    </scheduling-strategy>
+  </scheduler>
+  <!-- One or more processors here. -->
+</flow>
+----
++
+* The cron expression in this Scheduler triggers the generation of a Mule message in a new execution of the flow every fifteen seconds:
++
+[source, xml]
+----
+<flow name="ex-cron-expression-flow" >
+  <scheduler doc:name="Scheduler" >
+    <scheduling-strategy >
+      <cron expression='0/15 * * * * ? '/>
+    </scheduling-strategy>
+  </scheduler>
+  <!-- One or more processors here. -->
+</flow>
+----
 
-[%header,cols="1,2,1"]
+At least one processor must follow the Scheduler. Processors include Mule components and connector operations, such as an HTTP Request operation or Transform Message component. Failure to provide a processor produces a `MuleRuntimeException` and causes the deployment of the Mule application to fail with the following ERROR message in the logs (edited for readability):
```

Comment:
Not sure about including this here, this is for every flow (you cannot have an empty flow).

Reply:
Ok, it could go in the flow docs if it applies generally to all flows that lack processors after the message source.

Reply:
For context, I was hoping to show where the Scheduler fits in the flow architecturally so that people can understand the context. I'm not sure that the info hurts here, since this is the only Mule component that is a message source.

Reply:
We could move it to the flow doc later or share it there with an include.